src/app/users/views/organisations.ejs

```diff
@@ -86,6 +86,7 @@
                     </td>
                 <% } %>
                 <% if (org.requestDate) { %>
+                <td> </td>
```

Comment:
Does it need the space here between `<td> </td>`, as in making it `<td></td>` instead? 😄 If it needs the space then don't worry about it 👍